src/org/loklak/api/search/SearchServlet.java

```diff
@@ -184,7 +187,7 @@ public void run() {
                         DAO.log(request.getServletPath() + " scraping with query: " + scraper_query);
                         Timeline twitterTl = DAO.scrapeTwitter(post, scraper_query, order, timezoneOffsetf, true, timeout, true);
                         count_twitter_new.set(twitterTl.size());
-                        tl.putAll(QueryEntry.applyConstraint(twitterTl, tokens, false)); // pre-localized results are not filtered with location constraint any more 
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.